README.md

```diff
@@ -86,7 +86,7 @@ go build -o $GOPATH/bin/iop ./cmd/iop.go
 The `iop` command supports the following flags:
 
 - `logtostderr`: log to console (by default logs go to ./iop.go).
-- `dry-run`: console output only, nothing applied to cluster or written to files (default is true for now).
+- `dry-run`: console output only, nothing applied to cluster or written to files (**default is true for now**).
```

Comment:
s/for now//

Reply:
I want to emphasize this is temporary - reworded.